via/templates/pdf_viewer.html.jinja2

```diff
@@ -10,12 +10,22 @@
       the rest of via, however, we serve it with the PDF.js viewer application.
     -#}
     <link rel="canonical" href="{{ url }}"/>
+
+    <link rel="preload" as="fetch" href="{{ pdf_url }}" crossorigin>
+    <link rel="preload" as="worker" href="../build/pdf.worker.js">
+    <link rel="preload" as="script" href="{{ client_embed_url }}">
+
+    <link rel="icon" href="{{ static_url("via:static/favicon.ico") }}" type="image/x-icon" />
```

Comment:
I'm not quite sure why favicon has moved?

Reply:
Yep, I'm not sure either.